JS_HW_1/js_hw_1_task_2.js

```diff
@@ -0,0 +1,20 @@
+//  Решить квадратные уравнения. Переменные называть по правилам.
+// Вывести в консоль ответы в виде "Ответ к уравнению 1: <корень>", "Ответ к уравнению 2: <корень> и <корень>"
+// 1.  x2 - 6x + 9 = 0. - один корень
+// 2.  x2 - 4x - 5 = 0. - два корня
+
+let a1 = 1;
```

Comment:
I believe almost all variables in your code should be created with const, not let) except variables that can be changed